openwisp_radius/templates/radius_accounting_start.html

```diff
@@ -0,0 +1,5 @@
+{% load i18n %}{% load l10n %}
+<div class="msg">
+<p>{% trans "New session has been started for your account with username:" %} {{ user.get_username }}</p>
+<p>{% trans "You can manage your account or terminate this session any time by clicking on the button below." %}</p>
```

Comment:
You can review your session to find out how much time and/or traffic has been used or you can terminate the session at any time by clicking on the button below.